sdk/src/operator/hypercerts-storage.ts

```diff
@@ -1,48 +1,20 @@
-// @ts-ignore
-import { NFTStorage, CIDString } from "nft.storage";
-// @ts-ignore
-import { Web3Storage } from "web3.storage";
-import {
-  defaultNftStorageClient,
-  defaultWeb3StorageClient,
-  storeMetadata,
-  getMetadata,
-  storeData,
-  getData,
-} from "./index.js";
+import { storeMetadata, getMetadata, storeData, getData } from "./index.js";
 import { HypercertMetadata } from "../types/metadata.js";
 
-export interface HypercertsStorageConfig {
-  nftStorageToken?: string;
-  web3StorageToken?: string;
-}
-
 export class HypercertsStorage {
-  private nftStorageClient: NFTStorage;
-  private web3StorageClient: Web3Storage;
-
-  constructor(private config: HypercertsStorageConfig) {
-    this.nftStorageClient = config.nftStorageToken
```

Comment:
Is there a reason you didn't want to allow passing in tokens explicitly vs via environment variables? I think it's pretty helpful to make it explicit, especially for libraries, since otherwise it's easy to overlook that you need to set them

Reply:
For this specific file it seems redundant if we already do this in the index.ts in the same folder. But my goal primarily is to set this up using env vars and start adding in features for now. Generally, I think env vars are a cleaner practice and the app simply won't start if they're missing. If we accept people passing in `""` that could result in unexpected breaking stuff

Reply:
We can always make that nicer or more flexible in the future